back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/comment_on_idea_you_follow_mailer_preview.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module EmailCampaigns
+  class CommentOnIdeaYouFollowMailerPreview < ActionMailer::Preview
+    include EmailCampaigns::MailerPreviewRecipient
+
+    def campaign_mail
+      comment = Comment.where(post_type: 'Idea').first || Comment.create(post: Idea.first, author: User.first, body_multiloc: { 'en' => 'I agree' })
```

Comment:
Just checking, these previews are only internal aren't they? In which case it's fine to create a comment.

Reply:
Actually this is a very good point... I think somewhere they might be used, I'll check. The issue is that the seed file doesn't always create a comment, and we agreed in a code discussion meeting that we would generate the previews in this way instead of generating the event JSON (which would solve the issue in this case).

I think the right solution would be a refactoring of the seed data that is being generated. In this case, I think we could ensure that there is always a comment on an idea with a predetermined ID. This would also improve the quality of the previews (no lorem ipsum).